amundsen_application/models/user.py

```diff
@@ -1,15 +1,30 @@
-from typing import Dict
+from typing import Dict, Optional
 
 from amundsen_common.models.user import UserSchema, User
 from flask import current_app as app
 from marshmallow import ValidationError
 
 
+def _str_no_value(s: Optional[str]) -> bool:
```

Comment:
maybe a github issue todo to remove this workaround once metadata user object includes profile url as well?
cc @verdan

Reply:
Will make sure we have an issue